### ISSUE TITLE
Installation Guide - README.md

### ISSUE THREAD

**Issue by TheYellowArchitect** [supertuxkart/stk-code#4593]
I have been using linux daily for like a month now. Learnt vim and a lot of other complex stuff this way

I found [the download link](https://github.com/supertuxkart/stk-code/releases/tag/1.3-rc1) via https://blog.supertuxkart.net/2021/08/supertuxkart-13-release-candiate.html

And I read the README.md so I wouldn't fail the installation&execution.

But I didn't know how to run executables inside a folder. I was used to dumping them on the bin folder, and then I can just type them in the terminal. I couldn't drag the `run_game.sh` outside. I didn't know about `./run_game.sh` command... And I literally went and changed $PATH and included the entire supertuxkart1.3 folder (every file location is included, tons of bloat -_-), so as the terminal recognizes `run_game.sh`

tl;dr: I did something stupid, and I'm not the only one, or more accurately, I won't be the only one. Please add 1 line saying how to run it on the README.md, since more and more linux users are fresh off the windows train

**Comment by Benau** [comment]
I don't think README.md is the correct place, instead i updated https://github.com/supertuxkart/stk-code/releases/tag/1.3-rc1 so it tells user how to use the installer-less archives for linux and windows ![](https://cdn.lihkg.com/assets/faces/mouse/yea.gif)
